build-system/tasks/runtime-test/runtime-test-base.js

```diff
@@ -90,7 +90,15 @@ function updateBrowsers(config) {
       },
     })
     .set('edge', {browsers: ['Edge']})
-    .set('firefox', {browsers: ['Firefox']})
+    .set('firefox', {
+      browsers: ['FirefoxHeadless'],
+      customLaunchers: {
+        FirefoxHeadless: {
+          base: 'Firefox',
+          flags: ['-headless'],
```

Comment:
Instead of forcing Firefox to use headless mode, can we have `--headless` to conditionally add `flags: ['-headless']` to the browser config?

Reply:
Good point. Done.